examples/seq2seq/seq2seq.py

```diff
@@ -217,7 +221,9 @@ def main():
     optimizer = chainer.optimizers.Adam()
     optimizer.setup(model)
 
-    train_iter = chainer.iterators.SerialIterator(train_data, args.batchsize)
+    train_iter = chainer.iterators.SerialIterator(train_data,
+                                                  args.batchsize,
+                                                  shuffle=False)
```

Comment:
I would like to set random seed by numpy to avoid undeterministic behavior for mini-batches. In `__init__` of `SerialIterator`, `numpy.random.permutation` is called.